### PR TITLE
Pushing sduskis/v2_master to cloud-bigtable-client/v2_master

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/v2/BigtableDataGrpcClient.java [hunk 1]
```diff
@@ -342,6 +342,10 @@ public ReadModifyWriteRowResponse readModifyWriteRow(ReadModifyWriteRowRequest r
         new StreamObserverAdapter<>(readRowsCall, rowMerger);
     asyncUtilities.asyncServerStreamingCall(readRowsCall, request, listener);
 
+    if (request.getRows().getRowKeysCount() > 1 || request.getRows().getRowRangesCount() > 0) {
+      readRowsCall.request(4);
```
Comment (garye) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/v2/BigtableDataGrpcClient.java]: Where does 4 come from?

File: bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestBigtableSession.java [hunk 2]
```diff
@@ -28,17 +28,15 @@
 public class TestBigtableSession {
 
   private static final String PROJECT_ID = "project_id";
-  private static final String ZONE_ID = "zone_id";
-  private static final String CLUSTER_ID = "cluster_id";
+  private static final String INSTANCE_ID = "zone_id";
```
Comment (garye) [inline bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestBigtableSession.java]: zone_id -> instance_id

File: bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java [hunk 3]
```diff
@@ -44,16 +44,15 @@
     return null;
   }
 
-  public static Configuration configure(String projectId, String zoneName, String clusterName) {
+  public static Configuration configure(String projectId, String isntanceId) {
```
Comment (garye) [inline bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java]: typo: isntanceId

File: bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java [hunk 4]
```diff
@@ -485,24 +484,42 @@ public boolean isTableAvailable(TableName tableName) throws IOException {
...
-    CreateColumnFamilyRequest.Builder createColumnFamilyBuilder =
-        CreateColumnFamilyRequest.newBuilder()
-            .setColumnFamilyId(column.getNameAsString())
-            .setColumnFamily(columnFamily);
+  @Override
+  public void deleteColumn(TableName tableName, byte[] columnName) throws IOException {
+    final String columnNameStr = Bytes.toString(columnName);
+    Modification.Builder modification = Modification.newBuilder().setId(columnNameStr)
+        .setDrop(true);
+    modifyColumn(tableName, columnNameStr, "update", modification);
```
Comment (garye) [inline bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java]: update -> delete

File: bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/v2/BigtableDataGrpcClient.java (hunk 1 above)
Comment (sduskis) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/v2/BigtableDataGrpcClient.java]: it was an experiment.  I removed this change.

**Comment by sduskis** [comment]
I addressed the comments.  The typos were fixed and the experiments removed.  PTAL

**Comment by garye** [comment]
LGTM

**Pull request merged**